### PR TITLE
feat: expose pubsub interface

### LINKED ISSUE #258: Question: Access pubsub outside of resolvers
Hey, is there a way to access the pubsub from outside of any resolver function to publish events?

### PR DESCRIPTION
**Pull request by PacoDu**
Fixes #258

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -119,6 +119,7 @@ const plugin = fp(async function (app, opts) {
 
   if (subscriptionOpts) {
     subscriber = new PubSub(emitter)
+    app.decorate('pubsub', subscriber)
```
Comment (mcollina) [inline index.js:122]: This should be added to the `fastifyGraphql` object like https://github.com/mcollina/fastify-gql/pull/261/files#diff-168726dbe96b3ce427e7fedce31bb0bcR230.

File: test/subscription.js [hunk 2]
```diff
@@ -840,3 +840,101 @@ test('subscription server sends correct error if execution throws', t => {
     })
   })
 })
+
+test('subscription server exposes pubsub', t => {
+  // t.plan(2)
```
Comment (mcollina) [inline test/subscription.js:845]: why is this commented out?

Comment (PacoDu) [reply test/subscription.js]: This is not required, the test is working without t.plan

**Review by mcollina — APPROVED** [review]
lgtm

**Comment by PacoDu** [comment]
Can we bump the version pls ? :rocket:

**Pull request merged**